packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/SendCryptoInput/SendCryptoSelect/index.tsx

```diff
@@ -48,7 +48,12 @@ const SendCryptoSelect = () => {
                         setValue(CRYPTO_INPUT, '');
                         setValue(FIAT_INPUT, '');
                         const token = selected.value;
-                        if (token === 'ETH' || token === 'TGOR' || token === 'ETC') {
+                        if (
+                            token === 'ETH' ||
+                            token === 'TSEP' ||
+                            token === 'TGOR' ||
+                            token === 'ETC'
```

Comment:
`['ETH', 'TSEP', 'TGOR', 'ETC'].includes(token)` and possibly use the network config to pull all `networkType: 'ethereum'` token names dynamically, especially considering it's used in multiple places.

Reply:
d159ece4d3309f70414e05e094d148f670899429